bios/stage-2/src/main.rs

```diff
@@ -98,6 +99,17 @@ fn start(disk_number: u16, partition_table_start: *const u8) -> ! {
     writeln!(screen::Writer, "loading kernel...").unwrap();
     let kernel_len = load_file("kernel-x86_64", KERNEL_DST, &mut fs, &mut disk, disk_buffer);
     writeln!(screen::Writer, "kernel loaded at {KERNEL_DST:#p}").unwrap();
+    writeln!(screen::Writer, "Loading ramdisk...").unwrap();
+    let ramdisk_len = match try_load_file("ramdisk", RAMDISK_DST, &mut fs, &mut disk, disk_buffer) {
```

Comment:
Instead of using a hardcoded address for the ramdisk, we could use `KERNEL_DST.wrapping_add(kernel_len)`, rounded up to the next mulitple of teh page size.